content/docs/introduction/billing-sample.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Sample project billing
+enableTableOfContents: true
+subtitle: Practical example of how Neon pricing is calculated
+---
+
+## Generative AI example
+
+To give you a clearer sense of how billing works for Neon Pro users, let's explore a real-world example. Consider a simple image generation app that leverages Neon as the serverless database for storing user authentication details as well as records of image generation per user. Analyzing this usage over a monthly billing period can help you understand the nuances of Neon billing based on actual scenarios.
+
+## Overview: Costs by usage
+
+Roughly six months since launch, this high-traffic application attracts about 80K visitors daily, up to 450K weekly. It receives a steady influx of new users, with 3-5 new accounts created every hour. Each user's activity is capped at 5 images per month. This pattern of interaction and account creation gives you a sense of the steady volume of activity hitting the database. 
+
+## Assumptions
+
+### Tech stack (user management portion of the app):
+* **Authentication**: [NextAuth.JS](https://next-auth.js.org/) for authentication with OAuth
+* **Database**: Neon Serverless PostgreSQL to store user info and session detail
+* **ORM**: [Prisma ORM](https://www.prisma.io/) for database interactions
+* **Deployment Region**: US East (Ohio)
+
+### Userbase:
+* **Daily Active Users.** 80,000 users/day, implying a consistent volume of read queries. With a global, consumer-oriented user base, traffic is evenly distributed with no distinct peaks or dormant periods.
+* **Account creation.** Average of 3-5 sign-ups per hour, totally 120 new accounts per day. This gives you an idea of the number of write operations to the database for user authentication.
+* **User activity.** Each user's usage is capped at 5 generations per month. This includes logging IDs of generated photos and the incremental number of generations, which are written to the relevant tables.
+
+<Admonition type="note">
+Given the high number of connections used by this application, [connection pooling](/docs/connect/connection-pooling) is essential. And since the database connection is managed through Prisma, the datbase connection string also requires the addition of `?pgbouncer=true` for proper performance (see [Use connection pooling with Prisma](/docs/guides/prisma#use-connection-pooling-with-prisma)). 
```

Comment:
```suggestion
Given the high number of connections used by this application, [connection pooling](/docs/connect/connection-pooling) is essential. And since the database connection is managed through Prisma, the database connection string also requires the addition of `?pgbouncer=true` for proper performance (see [Use connection pooling with Prisma](/docs/guides/prisma#use-connection-pooling-with-prisma)). 
```